gradle.properties

```diff
@@ -27,7 +27,7 @@ kover.enabled=true
 kotlin.mpp.enableCompatibilityMetadataVariant=true
 kotlin.mpp.stability.nowarn=true
 
-kotlin.js.compiler=both
+kotlin.js.compiler=ir
```

Comment:
This can be straight-up removed, right? IR is the default and so you don't need to specify this anymore.

Reply:
I guess so, but there's no harm in explicitly mentioning that

Reply:
I'd suggest removing it, it's really tough to keep track of all the properties later on